src/qibolab/platform.py

```diff
@@ -25,7 +27,7 @@ class Platform:
     """
 
     def __init__(self, name, runcard, instruments, channels):
-        log.info(f"Loading platform {name}")
+        log.info("Loading platform %s", name)
```

Comment:
Why this? Usually Pylint complaints of the opposite, that you should use f-strings when possible (even though, more than Pylint, it's pyupgrade the one complaining).

I'm just surprised, do you have a Pylint code for this?

Reply:
Looks like it's because the logger is able to lazy-evaluate the arguments https://stackoverflow.com/a/34634301/8653979
Not sure how much is useful, but that's interesting :D

(Interestingly enough, the Pylint page has also a *rational* link, pointing to that SO answer - thanks Pylint!)